suites/squid/rgw/tier-2_rgw_regression_extended.yaml

```diff
@@ -230,6 +230,25 @@ tests:
         script-name: test_bucket_lifecycle_object_expiration_transition.py
         config-file-name: test_lc_process_without_applying_rule.yaml
 
+  - test:
+      name: Test user creation with placement and storage class cold
+      desc: Test user creation with placement and storage class cold
+      polarion-id: CEPH-83575880
+      module: sanity_rgw.py
+      config:
+        script-name: user_create.py
+        config-file-name: test_user_with_placement_id_storage_class_cold.yaml
+
+  - test:
+      name: Test user creation with placement and storage class
+      desc: Test user creation with placement and storage class
+      polarion-id: CEPH-83575880
+      comments: known issue (BZ-2311474)
+      module: sanity_rgw.py
+      config:
+        script-name: user_create.py
+        config-file-name: test_user_with_placement_id_storage_class.yaml
```

Comment:
@ckulal 
why this testcase not added in reef?

Reply:
we have known issue (BZ-2311474) for this scenario which is currently targeted to squid,
once the fix is available we can backport BZ to reef.

For now it is sure that issue present in reef as well, so well keep testcase only for squid for now

Reply:
okay @ckulal 
thanks for the clarity